app/src/main/java/app/grapheneos/camera/ui/SettingsDialog.kt

```diff
@@ -231,6 +238,24 @@ class SettingsDialog(val mActivity: MainActivity) :
                 override fun onNothingSelected(p0: AdapterView<*>?) {}
             }
 
+        videoFrameRateSpinner = binding.videoFrameRateSpinner
+
+        videoFrameRateSpinner.onItemSelectedListener =
+            object : AdapterView.OnItemSelectedListener {
+                override fun onItemSelected(
+                    p0: AdapterView<*>?,
```

Comment:
Proper parameter names are missing, likely due to lack of Android SDK sources. Check whether they are downloaded in Android Studio SDK manager

Reply:
For some reason, Android Studio wasn't checking the sources of SDK 33 that were already present, but adding the sources of SDK 34 did help resolve this issue. Thanks for mentioning this out!